pkg/rtc/mediaengine.go

```diff
@@ -11,6 +10,7 @@ import (
 )
 
 var opusCodecCapability = webrtc.RTPCodecCapability{MimeType: webrtc.MimeTypeOpus, ClockRate: 48000, Channels: 2, SDPFmtpLine: "minptime=10;useinbandfec=1"}
+var redCodecCapability = webrtc.RTPCodecCapability{MimeType: sfu.MimeTypeAudioRed, ClockRate: 48000, Channels: 2, SDPFmtpLine: "111/111"}
```

Comment:
Is 111 going to work always? Is this dependent on what client uses for opus payload? Or a pure Pion dependency?

Reply:
For publisher, in `ParticipantImpl.setCodecPreferencesOpusRedForPublisher`, will use the real opus payload
```
		if strings.EqualFold(codec.Name, "red") && strings.Contains(codec.Fmtp, strconv.FormatInt(int64(opusPayload), 10)) {
			preferredCodecs = append(preferredCodecs, strconv.FormatInt(int64(codec.PayloadType), 10))
		} else {
			leftCodecs = append(leftCodecs, strconv.FormatInt(int64(codec.PayloadType), 10))
		}
```
For subscriber, the answer side should use server's payload type, so it is safe to use that.